.github/workflows/archery.yml

```diff
@@ -51,10 +53,12 @@ jobs:
           python-version: '3.7'
       - name: Install
         working-directory: dev/archery
-        run: pip install pytest responses ruamel.yaml toolz jinja2 -e .
+        run: pip install pytest python-dotenv responses ruamel.yaml toolz jinja2 -e .
```

Comment:
Why not use pip install -e to ensure that requirements are up to date?

Reply:
There is `-e .` at the and of the command. The additional dependencies listed here are required for testing and crossbow.